src/components/presentational/AnimatedRing/AnimatedRing.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { CSSProperties } from 'react'
+import './AnimatedRing.css'
+
+export interface AnimatedRingProps {
+    children: React.ReactNode;
+    style?: CSSProperties;
+    color?: string;
+}
+
+export default function (props: AnimatedRingProps) {
```

Comment:
We have used circular progress bars in a number of components...so wondering if we should instead:
- Call this "ProgressRing"
- Have it take a % fill
- Have an "animated" property

Could see that being useful more broadly

Reply:
Done.